src/clarity-angular/button/index.ts

```diff
@@ -4,7 +4,18 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 import {Type} from "@angular/core";
-import {ClrButton as Button} from "./button-group/button";
-import {ClrButtonGroup as ButtonGroup} from "./button-group/button-group";
+import {ClrButton} from "./button-group/button";
+import {ClrButtonGroup} from "./button-group/button-group";
 
-export const BUTTON_GROUP_DIRECTIVES: Type<any>[] = [ButtonGroup, Button];
+export * from "./button-group/button";
+export * from "./button-group/button-group";
+export const CLR_BUTTON_GROUP_DIRECTIVES: Type<any>[] = [ClrButtonGroup, ClrButton];
+
+/* tslint:disable variable-name */
+/** @deprecated since 0.11 */
+export const Button = ClrButton;
+/** @deprecated since 0.11 */
+export const ButtonGroup = ClrButtonGroup;
```

Comment:
You already do all this in `button-group/index.ts`, why are you doing it again here? Just the `export * from "./button-group/button-group";` is enough.

Reply:
Turns out this file is never imported, so it could be removed.

Reply:
Oh, main gets the submodules? We should make a tree of index files, should we?
Main imports from button which itself imports from button-group. That would make maintenance easier, otherwise we have two files to update every time.